src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,94 @@
 import React from 'react'
+import { Typography, TextField, Button, Grid } from '@mui/material';
+import styles from '../SignUp/SignUp.module.scss';
+import signUpImage from "../../assets/Rectangle 8.png"
+import { useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
 
 export default function ChangePassword() {
+
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit
+  } = useForm({ mode: "all" })
+
+  const onSubmit = (data) => {
+    console.log(data);
+    
+    axios.post("http://154.41.228.234:3000/api/v0/admin/users/change-password", data
```

Comment:
In the change password, we will need to send the token as well, as we are not allowed to change password if we are not signed in.

Reply:
this component will edit and moved to sidebar as popup after merge with main